code/controller/astar.cpp

```diff
@@ -394,7 +395,7 @@ array2d<int> nrg::grid_kernelize(
     weak_ref<GridDisplay> grid,
     weak_ref<param_manager> pm
 ) {
-    constexpr int wall = GridDisplay::DEFAULT_WEIGHT;
+    int wall = GridDisplay::default_weight();
```

Comment:
make `GridDisplay::default_weight()` a `constexpr` function and add all the `constexpr` back

Reply:
Aight if I do this then I need to put the function back in the `griddisplay.h` file, but since we moved the enums, I can't access the constants from the header. Should I move the enum back?

Reply:
ehh you're right forget about it